app/helpers/posts_helper.rb

```diff
@@ -0,0 +1,19 @@
+module PostsHelper
+
+	# Determin which posts are relevant to the user
+	def get_relevant_posts
+		Post.tagged_with(current_user.tag_list, any: true).order('updated_at DESC')
+	end
```

Comment:
again - this is a service, not a helper.

do some googling on how to use view helpers in rails